atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/keyvalue/api/cache/ValueStoreImpl.java

```diff
@@ -24,11 +24,12 @@
 import com.palantir.atlasdb.keyvalue.api.Cell;
 import com.palantir.atlasdb.keyvalue.api.CellReference;
 import com.palantir.atlasdb.keyvalue.api.TableReference;
+import com.palantir.atlasdb.keyvalue.api.watch.WatchedCellRanges;
+import com.palantir.atlasdb.transaction.api.RowReference;
```

Comment:
it's slightly odd that this isn't in the same package as `CellReference` and `TableReference`. Not sure why that is, but changing it would be an API break, and creating an identical value type sounds silly.